src/plugins/intel_gna/tests/unit/gna_mock_api_initializer.hpp

```diff
@@ -0,0 +1,23 @@
+// Copyright (C) 2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+#include "gna_mock_api.hpp"
+
+#include <gna2-common-api.h>
+
+#include "cstdint"
+#include "vector"
+
+class GnaMockApiInitializer {
+    GNACppApi _mock_api;
+    std::vector<std::vector<uint8_t>> _data;
```

Comment:
What is _data? A more informative name would be welcome.

Reply:
I hope `_mocked_gna_memory` would be fine